data_collection/processing/event_type_detection.rst

```diff
@@ -0,0 +1,151 @@
+.. _data-collection-processing-event-type-detection:
+
+Event type detection
+=================
+
+Available event types
+-----------------
+
+Analyzing various properties of the tracked event we recognize one of the following types:
+
+- PageView
+- Outlink
+- Download
+- Search
+- Custom
+- ContentImpression
+- ContentInteraction
+- GoalConversion
+- OrderCompleted
+- AbandonedCart
+- SharePoint
+- ConsentFormImpression
+- ConsentFormClick
+- ConsentDecision
+- CustomPing
+- CartUpdated
+- BrokenEvent
+- ExcludedEvent
+- Heartbeat
+- Deanonymization
+- PagePerformance
+
+Detection order
+-----------------
+
+This section describes the order of the event type detection. Process is satisfied with the first match, after that it stops and no further matching is performed.
+
+Conversion
+~~~~~~~~~~~~~~~~~~
+
+First step in the event type detection process is a check for conversion signs in the tracked event.
+If your tracked event had an `idgoal` parameter with the ID value other than `0` then it will be marked as `GoalConversion` type.
+
+Ping
+~~~~~~~~~~~~~~~~~~
+
+Then the `ping` parameter is analyzed. Depending on it's value one of the following ping types will be assigned:
+
+- values `1`, `2` and `3` result in `Heartbeat` ping type
```

Comment:
Looks good, but quotes using single "`" character in RST format are formatted as _italic_, if you want to use `code` format they should be double "``" character quotes.

Example:
```
... result in ``Heartbeat`` ping type
```

Here's a [preview](https://developers.piwik.pro/en/pptt-3647_change_event_detection_order/data_collection/processing/event_type_detection.html) of this branch.

Reply:
I was aiming at any kind of emphasis, italics looked good enough but indeed code styling looks better. I just thought that there's something wrong with the formatting and the code block just looks like a italic block :)
Thx for the tip.